packages/sdk/src/types.ts

```diff
@@ -57,21 +57,23 @@ export enum RPC_ENDPOINTS  {
 }
 
 
-export type MBJS_CONFIG_PARAMS = {
-  network: Network | string;
+export type ConfigOptions = {
+  network?: Network | string;
   contractAddress?: string;
   callbackUrl?: string;
+  apiKey?: string;
 }
 
-export interface MbJsConfigObj extends MBJS_CONFIG_PARAMS {
+export interface ConfigOptionsObj extends ConfigOptions {
```

Comment:
Generally kind of okay with `enum` being upper case... but I think we are using these as `const.  Unless we actually intend to make any of these items enumerable i.e. mappable via numeric index from database enum field, dropdown index or something can just keep them as const.  

May or may not address this here, if anyone loves enums strongly for other reasons I am not considering, please speak up.

Reply:
Going to leave this as is based on internal conversation